js/cmb2.js

```diff
@@ -792,6 +797,9 @@ window.CMB2 = window.CMB2 || {};
 			if ( $element.hasClass('cmb2-media-status') ) {
 				// special case for image previews
 				val = $element.html();
+			} else if ( $element.hasClass('cmb2-textarea-code') ) {
```

Comment:
Not all instances of `.cmb2-textarea-code` will have code mirror initiated for them.  See here: https://github.com/CMB2/CMB2/wiki/Field-Types#notes

Reply:
This has been addressed with an extra test.